notes.md

```
@@ -22,6 +22,23 @@ lib
+    - find_min_potential_derivative_in_window (required)
+    - input_eval (required)
+    - lin_jitter (required)
+    - optimize (implemented)
+    - neurons should have inputs as vec but the network has no connections
+
+    sim
+    - find_threshold_crossing
+
+alpha
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)